service/directory-upload.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+usage="
+usage: $(basename "$0") command [-h] [-s] [-d] arguments...
+Uploads Measure and Library resources to the measure repository service.
+Options:
+    -h
+        Displays help menu.
+    -s [server baseUrl]
+        Specifies the base URL of the FHIR server to access.
+    -d
+        Provides directory path to parse for upload.
+"
+
+while getopts ':h:s:d:' option;
+do
+  case "$option" in
+    h)
+       echo -e "$usage"
+       exit 0
+       ;;
+    s)
+       server=$OPTARG
+       ;;
+    d)
+       directory_path=$OPTARG
+       ;;
+
+   \?) printf "illegal option: -%s\n" "$OPTARG" 1>&2
+       echo "$usage" 1>&2
+       exit 1
+       ;;
+    : )
+      echo "Invalid option: $OPTARG requires an argument" 1>&2
```

Comment:
I hit this option only with a -h (no argument), which I'm assuming should just show the usage.